server/routes/bookings/bookingsController.test.ts

```diff
@@ -3,22 +3,25 @@ import request from 'supertest'
 import * as cheerio from 'cheerio'
 import { SessionData } from 'express-session'
 import { appWithAllRoutes } from '../testutils/appSetup'
-import { createMockBookerService, createMockVisitService } from '../../services/testutils/mocks'
+import { createMockVisitService } from '../../services/testutils/mocks'
 import TestData from '../testutils/testData'
 import paths from '../../constants/paths'
 
 let app: Express
 
-const bookerService = createMockBookerService()
 const visitService = createMockVisitService()
 const bookerReference = TestData.bookerReference().value
 
 let sessionData: SessionData
 
 beforeEach(() => {
-  sessionData = {} as SessionData
+  sessionData = {
```

Comment:
Actually no need to mock booker (or booker prisoners) for each test now as I added it to `server/routes/testutils/appSetup.ts` because we can always assume it's there (because of `populateCurrentBooker()` middleware) - unless handling a special case. I'll fix 👍